src/components/CurrencyInputPanel/CurrencyInputPanel.container.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useCallback, useEffect, useMemo } from 'react'
+import debounce from 'lodash.debounce'
+
+import { CurrencyInputPanelComponent } from './CurrencyInputPanel.component'
+
+import {
+  useCurrencySearchModalSwap,
+  useCurrencySearchModalBridge,
+} from '../SearchModal/CurrencySearchModal/CurrencySearchModal.hooks'
+import { useBridgeInputValidation } from '../../pages/Bridge/ActionPanel/useBridgeInputValidation'
+
+import { CurrencyWrapperSource } from '../CurrencyLogo'
+import { CurrencyInputPanelProps } from './CurrencyInputPanel.types'
+
+import { CurrencySearchModalProvider } from '../SearchModal/CurrencySearchModal'
+import { normalizeInputValue } from '../../utils'
+
+export const CurrencyInputPanel = (currencyInputPanelProps: CurrencyInputPanelProps) => {
+  const searchModalContexts = useCurrencySearchModalSwap()
+
+  return (
+    <CurrencySearchModalProvider {...searchModalContexts}>
+      <CurrencyInputPanelComponent {...currencyInputPanelProps} />
+    </CurrencySearchModalProvider>
+  )
+}
+
+export const CurrencyInputPanelBridge = (currencyInputPanelProps: CurrencyInputPanelProps) => {
```

Comment:
Please put the complete interface just above component. So we can know the types better when looking at this file alone.